CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Fix an issue, where id-reuse could execute a PostAction on a newly registered event source
+
```

Comment:
Could you put it in a "Bugfixes" subsection? To fit the organization of the previous releases.